openupgrade_scripts/scripts/event_sale/17.0.1.3/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,21 @@
+---Models in module 'event_sale'---
+---Fields in module 'event_sale'---
+event_sale   / event.event.ticket       / _order                        : _order is now 'event_id, sequence, price, name, id' ('event_id, price')
+# NOTHING TO DO
+
+event_sale   / event.event.ticket       / price_incl (float)            : NEW hasdefault: compute
+# NOTHING TO DO: not sure if needed to precreate it.
+
+event_sale   / event.registration       / is_paid (boolean)             : DEL
+event_sale   / event.registration       / sale_status (selection)       : NEW selection_keys: ['free', 'sold', 'to_pay'], isfunction: function, stored
```

Comment:
This one is a bit more critical, as the number of registrations for sure will be high, so you have to pre-create it and only assign when there are payed events.